test/functional/autocmd/cursormoved_spec.lua

```diff
@@ -31,4 +31,15 @@ describe('CursorMoved', function()
     eq({'aaa'}, funcs.nvim_buf_get_lines(eval('g:buf'), 0, -1, true))
     eq(0, eval('g:cursormoved'))
   end)
+
+  it("is not triggered by cursor movement prior to first CursorMoved instantiation", function()
+    source([[
+    " Disable loading matchparen to avoid masking the behavior by pre-defining a valid autocommand
+    let g:loaded_matchparen = 1
```

Comment:
I think this is too late, you need to use `clear{args={'--cmd', 'let g:loaded_matchparen = 1'}}` before source to disable it before plugin/ files are loaded.

Reply:
Oh really? Because it causes a failure on master, but not on this branch...

Reply:
Ah, thanks. resolved (I think).